assingment.playground/Contents.swift

```diff
@@ -2,5 +2,41 @@ import Foundation
 
 private let workplace = Workplace()
 
-workplace.setEmptySeat(count: <#Int#>)
-workplace.employeesInOffice(employeeCount: <#Int#>)
+workplace.setEmptySeat(count: 9)
+workplace.employeesInOffice(employeeCount: 6)
+
+public func isSeatAvailable(_ seatIndex: Int) -> Bool {
+    let seat = workplace.seats[seatIndex]
+    if seat == nil {
+        return true
+    } else {
+        return false
+    }
+}
+
+public func occupyEmptySeat(index: Int, employee: String = "👩🏻‍💼") {
+    if index >= 0 && index < workplace.seats.count {
```

Comment:
**Fun fact:** this line can be written shorter `if 0 ..< workplace.seats.count ~= index {`

_It validated the variable `index` is in range between 0 and `workplace.seats.count - 1`_
_Vai mainīgais `index` pieder kopai [0; x), kur x - `workplace` objekta, `seats` mainīgā elementu skaits_